src/pages/SignUp/SignUp.style.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from "@emotion/react";
+
+import { theme } from "@/styles/theme";
+
+export const SignUpStyle = css`
```

Comment:
여기도 마찬가지로 컴포넌트가 아닌 일반 변수니까 카멜 케이스를 따르는게 일관성 있을것 같습니다~
```suggestion
export const signUpStyle = css`
```